common/meta/app/src/share/share.rs

```diff
@@ -344,6 +365,20 @@ impl ShareMeta {
         self.accounts.remove(account);
     }
 
+    pub fn get_grant_entry(&self, object: ShareGrantObject) -> Option<ShareGrantEntry> {
+        self.database.as_ref()?;
+
+        let database = self.database.as_ref().unwrap();
```

Comment:
Can it be rewritten to this?
```suggestion
        let database = self.database.as_ref()?;
```